packages/malloy/src/dialect/trino/dialect_functions.ts

```diff
@@ -62,9 +145,117 @@ const string_agg_distinct: OverloadedDefinitionBlueprint = {
   },
 };
 
+const variance: DefinitionBlueprint = {
+  takes: {'value': {dimension: 'number'}},
+  returns: {measure: 'number'},
+  impl: {function: 'VARIANCE'},
+};
+
+// Scalar functions
+
+const bitwise_and: DefinitionBlueprint = {
+  takes: {'val1': 'number', 'val2': 'number'},
+  returns: 'number',
+  impl: {
+    sql: 'BITWISE_AND(${val1}, ${val2})',
+  },
+};
+
+const bitwise_or: DefinitionBlueprint = {
+  takes: {'val1': 'number', 'val2': 'number'},
+  returns: 'number',
+  impl: {
+    sql: 'BITWISE_OR(${val1}, ${val2})',
+  },
+};
+
+const date_format: DefinitionBlueprint = {
+  takes: {'ts_val': 'timestamp', 'format': 'string'},
+  returns: 'string',
+  impl: {
+    sql: 'DATE_FORMAT(${ts_val}, ${format})',
+  },
+};
+
+const date_parse: DefinitionBlueprint = {
+  takes: {'ts_string': 'string', 'format': 'string'},
+  returns: 'timestamp',
+  impl: {
+    sql: 'DATE_PARSE(${ts_string}, ${format})',
+  },
+};
+
+const from_unixtime: DefinitionBlueprint = {
+  takes: {'unixtime': 'number'},
+  returns: 'timestamp',
+  impl: {function: 'FROM_UNIXTIME'},
+};
+
+const json_extract_scalar: DefinitionBlueprint = {
+  takes: {'json_val': 'string', 'json_path': 'string'},
```

Comment:
Seems like this can be `'json_val': ['string', 'json']`. Looking at the docs, it seems like the function supports both a string or JSON as the first argument (e.g. `JSON_EXTRACT_SCALAR(JSON '{"a": 1, "b": 2}', '$.a')`)